code/_onclick/item_attack.dm

```diff
@@ -89,26 +89,34 @@
 		to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
 		return
 
-	var/bigleagues = 10 //flat additive
-	var/buffout = force*0.25
-	var/smutant = force*0.25
-	var/ghoulmelee = force*0.25 //negative trait, this will cut 25% of the damage done by melee
+	// This code used to modify the force of the weapon directly, which is bad.
+	// It also had a lot of code duplication (many if statements with 'force >= 5' in them).
+	// Let this be a lesson not to do that. When in doubt, remember:
+	// Don't Repeat Yourself!
 
-	//var/regular = force*(user.special_s/100)//SPECIAL integration
+	//damage_multiplier += (user.special_s/100)//SPECIAL integration
 
-	//force += regular//SPECIAL integration
+	var/bonus_damage = 0
+	if (force >= 5)
+		if(istype(src, /obj/item/melee/unarmed))
+			if(HAS_TRAIT(user, TRAIT_UNARMED_WEAPON) && iscarbon(user))
+				var/mob/living/carbon/carbon_user = user
+				if(carbon_user.gloves == src) // if worn
+					bonus_damage += carbon_user.dna.species.punch_damage_roll(user, M)
+		else if(HAS_TRAIT(user, TRAIT_BIG_LEAGUES))
+			bonus_damage += 10 //flat additive
 
-	if (force >= 5 && HAS_TRAIT(user, TRAIT_BIG_LEAGUES))
-		force += bigleagues
+		if(HAS_TRAIT(user, TRAIT_BUFFOUT_BUFF))
+			damage_multiplier += 0.25
```

Comment:
```suggestion
			damage_multiplier += 0.4
```
We can afford Buffout to be strong since it's relatively short lasting and the punishment for OD is very intense.